### PR TITLE
Don't display a stack trace in the logs when .testcontainers.properties is not found

### PR DESCRIPTION
**Pull request by rishumehrotra**

### CODE REVIEW

File: core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java [hunk 1]
```diff
@@ -169,7 +169,7 @@ private static Properties readProperties(URL url) {
         try (InputStream inputStream = url.openStream()) {
             properties.load(inputStream);
         } catch (FileNotFoundException e) {
-            log.trace("Testcontainers config override was found on {} but the file was not found", url, e);
+            log.warn("Testcontainers config override was found on {} but the file was not found", url, e);
```
Comment (bsideup) [inline core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java]: this does not seem to fix the issue described in #2253.
If I understood it correctly, @vmassol was talking about **not** displaying the stacktrace but the only the error message, to make the logs less verbose

Comment (rishumehrotra) [reply core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java]: ugh. My bad. removing "e" should help then ?

Comment (bsideup) [reply core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java]: please follow the issue's description, it is very well described and suggests a solution

Comment (rishumehrotra) [reply core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java]: Understood. Will do.

Comment (rishumehrotra) [reply core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java]: Resubmitted

File: core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java [hunk 2]
```diff
@@ -169,9 +170,9 @@ private static Properties readProperties(URL url) {
         try (InputStream inputStream = url.openStream()) {
             properties.load(inputStream);
         } catch (FileNotFoundException e) {
-            log.trace("Testcontainers config override was found on {} but the file was not found", url, e);
+            log.warn("Testcontainers config override was found on {} but the file was not found", ExceptionUtils.getRootCauseMessage(e));
```
Comment (rnorth) [inline core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java:173]: I could be reading it wrong, but won't this insert the exception message right into the middle of the log string in a very strange way?

e.g. if the exception message is `something went wrong` then we're going to end up logging:

`Testcontainers config override was found on something went wrong but the file was not found`

I think we want:
```suggestion
            log.warn("Testcontainers config override was found on {} but the file was not found. Exception message: {}", url, ExceptionUtils.getRootCauseMessage(e));
```

and similar for the `IOException` case...

WDYT?

Comment (rishumehrotra) [reply core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java]: Thanks @rnorth . Fixing.

**Review by rnorth — APPROVED** [review]
Thanks @rishumehrotra - this looks good to me. Thanks for updating.

**Pull request merged**